backstop/backstop.js

```diff
@@ -30,7 +31,7 @@ const config = {
   debug: false,
   debugWindow: false,
   dockerCommandTemplate:
-    'docker run --rm -i --mount type=bind,source="{cwd}",target=/src backstopjs/backstopjs:{version} {backstopCommand} {args}',
+    'docker run --rm -i --mount type=bind,source="{cwd}",target=/src itwinui/backstopjs:latest {backstopCommand} {args}',
```

Comment:
this `latest` here refers to the version of the docker image. i'm setting `latest` in the `build-docker` command, so that's what I'm using here.

I think backstop will just use whatever version is installed.

@gretanausedaite Do you remember how this works? Because I'm still not sure where the `BACKSTOPJS_VERSION` environment variable comes from.

Reply:
So if we don't pass the environment variable, should I hardcode 6.0.4 in the dockerfile?

Reply:
then hardcode it in dockerfile.

Reply:
backstop version is now hardcoded in dockerfile